File-Formats/DocIO/Working-with-Footnotes-and-endnotes.md

```diff
@@ -700,6 +700,208 @@ using (WordDocument document = new WordDocument())
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/Footnotes-and-Endnotes/Change-default-endnote-separator).
 
+## Modify Footnote and Endnote content
+Modify footnote and endnote contents in an existing Word document.
+
+The following code example shows how to modify the footnote and endnote content from an existing word document.
+
+{% tabs %} 
+
+{% highlight c# tabtitle="C#" %}
+//Load an existing Word document.
+using (WordDocument document = new WordDocument("Input.docx", FormatType.Docx))
+{
+//Access paragraph in a Word document.
+WParagraph paragraph = document.Sections[0].Paragraphs[6] as WParagraph;
```

Comment:
@sharonaumao - Please align the code snippets properly as similar to below UG topic,
https://github.com/syncfusion-content/fileformat-docs/edit/master/File-Formats/DocIO/mail-merge/mail-merge-options.md

Reply:
@Ramaraj-Marimuthu -Done